src/controller/BackPostController.php

```diff
@@ -69,29 +56,49 @@ public function addPost(PostModel $postModel)
 			'id_user' => $id_user
 		];
 
+		// insert post if control ok
+		if (empty($errors)) {
+			$post = new Post;
```

Comment:
```suggestion
			$post = new Post;
			$post
			->setTitle($title)
			->setChapo($chapo)
		        ->setContents($contents)
			->setPublish('waiting')
			->setUser($id_user);
```
Pour que celà fonctionner il faut que tes setters retourn ton objet :
ex :  function setPublish(string $publish) : Post { $this->publish =  $publish; return $this;}

Reply:
Modif ok